src/test/ui/suggestions/integer-into.rs

```diff
@@ -0,0 +1,17 @@
+fn main() {
```

Comment:
Is there an existing test with int -> int conversions that these naturally belong to? If not, can you add a comment with the rationale for the test? A link to the issue would suffice.

```suggestion
// https://github.com/rust-lang/rust/issues/71580

fn main() {
```